plugin.py

```diff
@@ -743,7 +743,9 @@ def run(self, **kwargs) -> None:
             return
         env_path = kwargs.get('env_path')
         if env_path == SELECT_FOLDER_DIALOG_FLAG:
-            sublime.select_folder_dialog(self.on_select_folder, multi_select=False)  # pyright: ignore
+            starting_dir = os.path.dirname(self.window.active_view().file_name())
```

Comment:
This would raise an error if the active view is an unsaved buffer (`file_name()` is `None`).

Reply:
If the current view is unsaved the LSP shoulnd't even be enabled no?

Reply:
also, the directory in the `select_folder_dialog` method is by default `None`, so even with an unsaved view will just default to the current behaviour.

Reply:
This is a WindowCommand, and it is active as long as there is a Julia file in the current window. So you can switch the environment even if you're currently looking at another file type, e.g. Project.toml or something else.

The problem is that `os.path.dirname(None)` still produces a runtime error.